app/assets/stylesheets/application.scss

```diff
@@ -147,3 +154,610 @@ ul > li > ul > li {
 .white-space-pre-wrap {
   white-space: pre-wrap;
 }
+
+$dfe-blue: #003a69;
+$accent: #347ca9;
+
+.dfe-template__body {
+  .govuk-header {
+    background-color: $dfe-blue;
+  }
+
+  .govuk-header__container {
+    position: relative;
+    margin-bottom: -10px;
+    padding-top: 25px;
+    padding-bottom: 15px;
+    border-bottom: 10px solid $accent;
+  }
+
+  .govuk-header__product-name,
+  .govuk-header__logotype_text,
+  .govuk-header__content {
+    line-height: 2.5rem !important;
+    font-size: 1.2rem;
+  }
+
+  .govuk-header__logotype-crown-fallback-image {
+    height: 45px;
+    width: 45px;
+  }
+
+  .govuk-header__logotype {
+    margin-right: 0;
+  }
+
+
+  .govuk-header__logo {
+    @include mq ($from: desktop) {
+      width: 100%;
+    }
+  }
+
+  .govuk-header__content {
+
+    width: 100%;
+    float: none;
+
+  }
+
+  .govuk-header__content {
+    @include mq ($from: desktop) {
+      width: 100%;
+      float: right;
+    }
+  }
+
+
+  .govuk-header nav {
+    @include mq ($from: desktop) {
+      display: inline-block;
+      float: right;
+    }
+  }
+}
+
+.course-creator-image {
+  width: 90px;
+  height: 90px;
+}
+
+.align-right {
+  float: right;
+}
+
+.gem-c-feedback {
+  background: #fff;
+  margin-top: 30px;
+}
+
+.gem-c-feedback__prompt {
+  background-color: #1d70b8;
+  color: #fff;
+  max-width: 600px;
+  margin: 0 auto;
+}
+
+.gem-c-feedback__prompt-questions {
+  text-align: center;
+  border-bottom: 1px solid #fff;
+  padding: 15px 25px 0;
+  box-sizing: border-box;
+}
+
+.gem-c-feedback__prompt-question {
+  vertical-align: top;
+  display: inline-block;
+  margin: 10px 20px;
+  font-weight: 700;
+  color: govuk-colour("white");
+}
+
+.gem-c-feedback__option-list {
+  display: inline-block;
+  margin-right: 10px;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  margin-top: 10px;
+}
+
+.gem-c-feedback__prompt-link {
+  position: relative;
+  font-family: "GDS Transport", arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 16px;
+  font-size: 1rem;
+  line-height: 1.25;
+  background: transparent;
+  box-shadow: 0 2px 0 #fff;
+  border: 1px #fff solid;
+  min-width: 100px;
+}
+
+@media only screen and (max-width: 350px) {
+  .gem-c-feedback__prompt-link {
+    min-width: 10px;
+  }
+}
+
+.BGstrip-grey {
+  background-color: #f3f2f1;
```

Comment:
I think we should be consistent and continue to use the colours by name, is this an existing or new one?